api/office/schema.py

```diff
@@ -19,11 +19,17 @@
 
 
 class Office(SQLAlchemyObjectType):
+    """
```

Comment:
@TeamoreA this is good work though I am curious why we still have the `office/schema.py` in the v2 codebase

Reply:
It's already in v2. I got it after branching off v2